src/test_l1_dsDisplay.c

```diff
@@ -984,7 +984,7 @@ void test_l1_dsDisplay_positive_dsRegisterDisplayEventCallback(void) {
     // Step 01: Initialize the display sub-system
     result = dsDisplayInit();
     UT_LOG("\n In %s Return value: [%d]\n", __FUNCTION__, result);
-    UT_ASSERT_EQUAL(result, dsERR_NONE);
+    UT_ASSERT_EQUAL_FATAL(result, dsERR_NONE);
 
     // Step 02: Loop through all valid ports in kSupportedPortTypes
     for (int i = 0; i < sizeof(kSupportedPortTypes) / sizeof(kSupportedPortTypes[0]); i++) {
```

Comment:
Can we use the braces ( sizeof(kSupportedPortTypes) / sizeof(kSupportedPortTypes[0]) ) to have better operator precedence?